core/src/main/java/com/klaviyo/coresdk/networking/requests/NetworkRequest.kt

```diff
@@ -1,18 +1,19 @@
-package com.klaviyo.coresdk.networking
+package com.klaviyo.coresdk.networking.requests
```

Comment:
I moved all request types and abstract classes into a sub-folder since the networking code was expanding more and more. Hence the imports updated